app/graphql/inputs/enabled_dynamic_field_input.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Inputs
+  class EnabledDynamicFieldInput < Types::BaseInputObject
+    description 'Attributes for an enabled dynamic field in a project for a digital obect type'
+
+    argument :dynamic_field, Inputs::IdReference, required: true
```

Comment:
Is it possible to do `argument :dynamic_field, ID, required: true` instead of the above `Inputs::IdReference`?  (And same question for `field_sets` below.)

Reply:
Though with my suggestion, that may involve changing the name of the arguments to `dynamic_field_id` and `field_set_ids`, if we were passing IDs instead of objects.  But if we're only ever passing an object with one id attribute, would it be worth just naming the fields to indicate that they only accept IDs?  And then just use IDs?

Reply:
Ah, I see a pattern with the `Project::StringKey` input object.

Reply:
Good pattern!  Withdrawing my suggestion!